src/me/remie/vulcan/leaguetasks/task/tasks/fishing/FishAnchovies.java

```diff
@@ -0,0 +1,183 @@
+        super(script, "Catch an Anchovy");
+        this.fishingNetShopHelper = new ShopHelper(this);
+    }
+
+    @Override
+    public void run() {
+        // Validate we got everything before continue
+        if (!doWeHaveRequiredItem()) {
+            if (!teleportHome()) {
+                return;
+            }
+            if (!getRequiredItem()) {
+                ctx.log("Issues getting required items.");
+                return;
+            }
+            return;
+        }
+
+        // Teleport to Catherby
+        if (!areWeInCatherbyRegion()) {
+            if (!teleportHome()) {
+                return;
+            }
+            teleportToCatherby();
+            return;
+        }
+
+        // Navigate to bank and deposit inventory
+        if (areWeInCatherbyRegion() && ctx.inventory.inventoryFull()) {
+            depositItemsInCatherbyBank();
+            return;
+        }
+
+        // Navigate to fishing spot
+        if (!fishingSpotArea.within()) {
+            navigateToFishingSpot();
+            return;
+        }
+
+        fishAnchovies();
+    }
+
+    private boolean teleportToCatherby() {
+        script.setScriptStatus("Teleport to Catherby");
+        if (ctx.teleporter.open()) {
+            ctx.teleporter.teleportStringPath("Cities", "Catherby");
+            ctx.onCondition(() -> ctx.pathing.regionLoaded(LeagueScriptConstants.CATHERBY_REGION_ID), 250, 10);
+            return true;
+        }
+        return false;
+    }
+
+    private boolean areWeInCatherbyRegion() {
+        final int playerCurrentRegion = ctx.players.getLocal().getLocation().getRegionID();
+        if (playerCurrentRegion != LeagueScriptConstants.CATHERBY_REGION_ID && playerCurrentRegion != FISHING_SPOT_REGION) {
+            return false;
+        }
+        return true;
+    }
+
+    private boolean openUpBank() {
+        if (!ctx.bank.bankOpen()) {
+            script.setScriptStatus("Opening up bank");
+            final SimpleNpc banker = ctx.npcs.populate().filter(LeagueScriptConstants.BANKER_NPC_ID, 1613).nextNearest();
+            if (banker != null) {
+                banker.menuAction("Bank");
+                return ctx.onCondition(ctx.bank::bankOpen, 250, 10);
+            }
+            return false;
+        }
+        return true;
+    }
+
+    private boolean depositInventory() {
+        if (!openUpBank()) {
+            return false;
+        }
+
+        if (!ctx.bank.bankOpen()) {
+            return false;
+        }
+
+        script.setScriptStatus("Depositing inventory");
+        ctx.bank.depositAllExcept(LeagueScriptConstants.COINS_ID, SMALL_FISHING_NET.getItemId());
+        ctx.sleep(1000);
+        script.setScriptStatus("Closing bank");
+        ctx.bank.closeBank();
+        ctx.sleep(1000);
+        return true;
+    }
+
+    private boolean getRequiredItem() {
+        if (!ctx.inventory.populate().omit(LeagueScriptConstants.COINS_ID).omit(SMALL_FISHING_NET.getItemId()).isEmpty()) {
```

Comment:
```suggestion
        if (!ctx.inventory.populate().omit(LeagueScriptConstants.COINS_ID, SMALL_FISHING_NET.getItemId()).isEmpty()) {
```

Reply:
Forgot to update this after my discovery yesterday. 🤦